### PR TITLE
[MODULAR] Port Tarkon (New Ghostrole/Space ruin) [Ready for Review/Testing]

### PR DESCRIPTION
**Pull request by Zenitheevee**
## About The Pull Request

"It was an early start of a company... "Tarkon Industries", with the first test of its prototype going to work on a designated asteroid. It looked promising, embedded well enough and after a few days, we got the first return of the driver for more materials... Things were going oh... so... well..."

tarkon

Port Tarkon is, To put it a bit bluntly, A ghost role that started as a small re-imagining of Charliestation. However, after a handful of ideas and some news, I moved it up to a new ghostrole in of itself. 

Having a rather large amount of need to, say, Gather more equipment or to gather help, There *Is* a ship it can use to move around its own Z and to the station, And people attending to the role are made aware of their need to not just jump ship. 

Lastly, It includes its own Turbine engine, in place of an SM. 

## How This Contributes To The Skyrat Roleplay Experience

Adds a 5-person ghost role area that is a spiritual reimagining of charlie, allowing a proper redesigning of its layout if people desire and gives a decent mining area already encasing the spawning area, with a good handful of minor deterrents to disuade spacetiders.

## Future Plans For Tarkon

- Once lore gets hit, Lore and shit will eventually get more work on it. Currently its just very vague, And i'll plan on changing that
- Getting a functioning financing account for the people to use and an accompanying Express console. (Or not, atleast not without removing the cargobay full of stuff)
- Future Expansion and maybe even a 'canonical' clean out w/ potential threat differences being noted
- Maybe some more exclusive items/research stuff. 

## Changelog

:cl:
add: Adds Port Tarkon, an abandoned asteroid-embedded mini-station over-ran with problems.
add: Added Abandoned Crew ghost roles to Inhabit Port Tarkon.
add: 1 unique gun that spawns in a pair in the security office.
/:cl:

### CODE REVIEW

**Comment by Zenitheevee** [comment]
right, With the advent of the large HD map picture i've redacted the large clusterheck of closeups
But leaving hte one below of the ship.

And a picture of the ship: 
image

**Comment by dawsonkeyes** [comment]
have you considered easing up on the carp or xenomorph numbers? that's a _lot_ of carp

**Comment by Zenitheevee** [comment]
> have you considered easing up on the carp or xenomorph numbers? that's a _lot_ of carp

Carps are *literally* the easiest things to kill, Even back on charlie with the shitty retro lasers being a one-shot enemy.

The M6 takes a few shots to kill them but they also have mags so you dont have to wuss out to charge.

File: modular_skyrat/modules/mapping/code/fluff.dm [hunk 1]
```diff
@@ -80,6 +86,73 @@
...
+		5 = "<span class='game say'><span class='name'>Officer ???</span> <span class='message'>sniffles, \"<span class=' '>... Overseer Tavus... I... did what I could for them... Asked the crew to board the cargo shuttle... Leave the main shuttle if the sleepers activated...</span>\"</span></span>",
+		6 = "<span class='game say'><span class='name'>Officer ???</span> <span class='message'>groans, \"<span class=' '>God... Those.. Things. Aliens... They got Tavus... Severed his leg clean off... Told him that.. We'd clean out the port... And he'd wake up in the trauma bay...</span>\"</span></span>",
+		7 = "<span class='game say'><span class='name'>Officer ???</span> <span class='message'>coughs then calmly states, \"<span class=' '>... If.. Anyone wakes up... If the Ensign... Is alive... They're in charge now... The.. The Tarkon Drill's designs are... In the solars room, in a hidden floorsafe... Abandon the port or.. Clear it out... But dont let those designs go abandoned..</span>\"</span></span>",
+		8 = "<span class='game say'><span class='name'>The universal recorder</span> <span class='message'>says, \"<span class='tape_recorder '>Recording stopped.</span>\"</span></span>"
```
Comment (jjpark-kb) [inline modular_skyrat/modules/mapping/code/fluff.dm]: do we not have a span_x for these?

File: modular_skyrat/modules/mapping/code/fluff.dm [hunk 2]
```diff
@@ -80,6 +86,73 @@
...
+		5 = "<span class='game say'><span class='name'>Scientist Arkus</span> <span class='message'>sniffles, \"<span class=' '>... Is there a problem? Its been dormant ever since we've been here, If it was going to come alive it would have done so while putting the tiling down. Besides... I got lunch to attend...</span>\"</span></span>",
+		6 = "<span class='game say'><span class='name'>Foreman Verok</span> <span class='message'>groans, \"<span class=' '>Right... Right I'll.. Get to work on it then... Just keep an ear out...</span>\"</span></span>",
+		7 = "<span class='game say'><span class='name'>Foreman Verok</span> <span class='message'>grumbles, \"<span class=' '>Right... Calm down, Verok... Place the floor safe, scoot the tile back in place... And afterwards grab some money from the one by the financing console under the table... I dont think Tavus will notice an extra few credits missing...</span>\"</span></span>",
+		8 = "<span class='game say'><span class='name'>The universal recorder</span> <span class='message'>says, \"<span class='tape_recorder '>Recording stopped.</span>\"</span></span>"
```
Comment (jjpark-kb) [inline modular_skyrat/modules/mapping/code/fluff.dm]: Same thing here as previous.

**Comment by jjpark-kb** [comment]
I am not too sure how I feel about the additional unique gun in a PR that is really meant for this ruin; though, I can overlook it as it is documented in the changelog.

File: modular_skyrat/modules/mapping/code/fluff.dm (hunk 2 above)
Comment (jjpark-kb) [reply modular_skyrat/modules/mapping/code/fluff.dm]: I guess we don't! Ignore my comment then for now.

File: modular_skyrat/modules/mapping/code/fluff.dm (hunk 1 above)
Comment (jjpark-kb) [reply modular_skyrat/modules/mapping/code/fluff.dm]: I guess we don't! Ignore my comment then for now.

**Review by jjpark-kb — APPROVED** [review]
As long as you have tested it and it works, I am fine with this.

**Comment by Zenitheevee** [comment]
> As long as you have tested it and it works, I am fine with this.

Everything works and functions, I've tested it thoroughly and I've made sure everything outside of the large empress nest is doable within the confines of one person.

File: modular_skyrat/modules/mapping/code/fluff.dm (hunk 1 above)
Comment (Zenitheevee) [reply modular_skyrat/modules/mapping/code/fluff.dm]: Yeah sadly pre-recorded tapes are jank as all get-out.

**Comment by OolongCow** [comment]
> have you considered easing up on the carp or xenomorph numbers? that's a _lot_ of carp

It's not really a concern. PKA one-shots them with infinite ammo, and the designated miner is the one who should be going out there anyway.

**Pull request merged**